src/components/Extra.vue

```
@@ -36,8 +44,20 @@
     <!-- --------------------------------------------------------------------------------------------- -->
     <!-- Message Display -->
     <div id="MessageDiv">
-        <div v-for="mes in useState().successMessages" class='alert alert-success' role='alert'> {{ mes }}</div>
-        <div v-for="error in useState().errorMessages" class='alert alert-danger' role='alert'> {{ error }}</div>
+        <div
+            v-for="mes in useState().successMessages"
+            class="alert alert-success"
+            role="alert"
+        >
```

ESLint

[error] Elements in iteration expect to have 'v-bind:key' directives.

(vue/require-v-for-key)